src/services/calendar/index.js

```diff
@@ -179,7 +188,10 @@ class Service {
 	create(data, params) {
 		const serviceUrls = this.app.get('services') || {};
 
-		const userId = (params.query || {}).userId || (params.account || {}).userId || params.payload.userId;
+		const userId =
```

Comment:
Codacy found an issue: [There should be no line break before or after '='.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045787)